internal/backends/gl/glcontext.rs

```diff
@@ -41,7 +48,10 @@ impl OpenGLContext {
                 let current_ctx = unsafe { not_current_ctx.make_current().unwrap() };
                 OpenGLContextState::Current(current_ctx)
             }
+            #[cfg(not(target_arch = "wasm32"))]
             state @ OpenGLContextState::Current(_) => state,
+            #[cfg(target_arch = "wasm32")]
+            state @ OpenGLContextState::Current { .. } => state,
```

Comment:
```suggestion
            state @ OpenGLContextState::Current { .. } => state,
```

you can use the `{ .. }` syntax to match a tuple, iirc.

otherwise, just  `state => state`

Reply:
Thanks, that's indeed much better.